### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @truebguillaume

### DISCUSSION THREAD

**Discussion by grinlemon** [heig-vd-dai-course/heig-vd-dai-course#37] [Show and tell]
https://github.com/truebguillaume/java-maven-dai

**Comment by ludelafo** [comment]
Thank you!

I would add a few words on what the project is about and how to build and run the app.

Regarding the GitHub template, please come to me during the course, I have a few thoughts to be aware with this approach.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
